tests/unit/encryption.spec.js

```diff
@@ -0,0 +1,11 @@
+require("./util");
+const { decryptAES, encryptAES } = require("../../src/js/shared/encryption");
+const defaultSettings = require("../resources/defaultSettings.json");
+
+describe("Encryption", () => {
+  it("Test encryptAES and decryptAES functions", async () => {
+    const encryptedText = encryptAES(JSON.stringify(defaultSettings), "password");
+    const decryptedText = decryptAES(encryptedText, "password");
+    expect(JSON.parse(decryptedText)).toStrictEqual(defaultSettings);
+  });
```

Comment:
The unit tests don't satisfy the requirements of the issue's description:

 * Implement an unit test for `encryptAES` than will encrypt `resources/defaultSettings.json` and check that the result is equal to `resources/encryptedSettings.json`.
 * Implement an unit test for `decryptAES` than will decrypt `resources/encryptedSettings.json` and check that the result is equal to `resources/defaultSettings.json`.

E.g.:

```javascript
const defaultSettings = require("../resources/defaultSettings.json");
const encryptedSettings = require("../resources/encryptedSettings.json");

describe("Encryption", () => {
  it("encryptAES()", async () => {
    const encryptedText = encryptAES(
      JSON.stringify(defaultSettings),
      "password"
    );

    expect(encryptedText).toEqual(encryptedSettings.encryptedData);
  });

  it("decryptAES()", async () => {
    const decryptedText = decryptAES(encryptedSettings.encryptedData, "password");
    expect(JSON.parse(decryptedText)).toStrictEqual(defaultSettings);
  });
});
```

Reply:
Yes, I should have mentioned that, I couldn't do that part because when encrypting we are adding salt part which is random numbers. So, even if the same data is encrypted the different result will be returned. So, instead I just encrypted and decrypted back to check that result is equal to original data